demo/scripts/controls/contextMenu/ResetListPlugin.ts

```diff
@@ -22,17 +23,35 @@ export default class ResetListPlugin implements ContextMenuProvider<ContextMenuI
 
         if (safeInstanceOf(node, 'HTMLLIElement')) {
             const list = this.editor.getElementAtCursor('ol,ul', node);
-            if (safeInstanceOf(list, 'HTMLOListElement') && list.start > 1) {
-                items.push({
-                    key: 'resetList',
-                    name: 'Reset list number',
-                    onClick: () => {
-                        this.editor?.addUndoSnapshot(() => {
-                            this.editor.focus();
-                            list.start = 1;
-                        }, ChangeSource.Format);
+            if (safeInstanceOf(list, 'HTMLOListElement')) {
+                items.push(
+                    {
+                        key: 'resetList',
+                        name: 'Reset list number',
+                        onClick: () => {
+                            this.editor?.addUndoSnapshot(() => {
+                                this.editor.focus();
+                                setOrderedListNumbering(this.editor, list, 1);
+                            }, ChangeSource.Format);
+                        },
                     },
-                });
+                    {
+                        key: 'setNumberingValue',
+                        name: 'Set Numbering Value',
+                        onClick: () => {
+                            this.editor?.addUndoSnapshot(() => {
```

Comment:
same here

Reply:
Added the editor.focus and the addUndoSnapshot to the function in this commit.
d48eaee